ui/src/app/routing/app-routing.tsx

```diff
@@ -279,7 +277,6 @@ export const AppRoutingComponent: React.FunctionComponent<RoutingProps> =
                   <AppRoute
                     path=''
                     exact={false}
-                    intermediaryRoute={true}
```

Comment:
AppRoute did not actually use this prop.

It's only set in 2 cases, and in both cases the child component does use it.